index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.0/css/bootstrap.min.css" integrity="sha384-9gVQ4dYFwwWSjIDZnLEWnxCjeSWFphJiwGPXr1jddIhOegiu1FwO5qRGvFXOdJZ4" crossorigin="anonymous">
+  <link rel="stylesheet" href="./css/main.css">
+</head>
+<body>
+<div class="container-fluid">
+  <article>
+
+  <!-- header-->
+    <section class="header">
+      <!-- navigation bar -->
+      <nav class="navbar navbar-expand-lg">
+        <a class="navbar-brand" href="#">
+          <div class="logo"></div>
```

Comment:
I think since this element has no children, you can just make it a self-enclosing element like: `<div />`

Reply:
HTML doesnt support making regular tags into self closing tags like this :(

Reply:
Ah darn. I've been writing too much React today 😜